packages/paste-website/src/component-examples/TopbarExamples.ts

```diff
@@ -1,222 +1,887 @@
+              <UserDialogUserInfo>
+                <UserDialogUserName>Name</UserDialogUserName>
+                <UserDialogUserEmail>email@email.com</UserDialogUserEmail>
+              </UserDialogUserInfo>
+              <UserDialogList {...userDialogList} aria-label="User menu actions">
+                <UserDialogListItem {...userDialogList}>
+                  <UserIcon decorative />
+                  User settings
+                </UserDialogListItem>
+                <UserDialogSeparator />
+                <UserDialogListItem {...userDialogList}>
+                  <ThemeIcon decorative />
+                  <Box width="100%" display="flex" justifyContent="space-between">
+                    Theme
+                    <Badge variant="decorative10" as="span" size="small">
+                      Light
+                    </Badge>
+                  </Box>
+                </UserDialogListItem>
+                <UserDialogListItem {...userDialogList}>
+                  <TranslationIcon decorative />
+                  <Box width="100%" display="flex" justifyContent="space-between">
+                    Language
+                    <Badge variant="decorative10" as="span" size="small">
+                      EN
+                    </Badge>
+                  </Box>
+                </UserDialogListItem>
+                <UserDialogSeparator />
+                <UserDialogListItem {...userDialogList} href="https://www.google.com">
+                  <LogOutIcon decorative />
+                  Log out
+                </UserDialogListItem>
+              </UserDialogList>
+            </UserDialog>
+          </UserDialogContainer>
+        </TopbarActions>
+      </Topbar>
   )
 }
 render(
-  <TopbarExample/>
+    <TopbarExample/>
 )
 `.trim();
 
-export const emptyTopbarExample = `
+export const flexTopbarExample = `
 const TopbarExample = () => {
-    return (
-        <Topbar>I am an empty Topbar</Topbar>
-    )
+  const ConnectivityObject = {
+    Available: {
+      variant: 'ConnectivityAvailable',
+      children: 'Available',
+    },
+    Busy: {
+      variant: 'ConnectivityBusy',
+      children: 'Busy',
+    },
+    Unavailable: {
+      variant: 'ConnectivityUnavailable',
+      children: 'Unavailable',
+    },
+    Neutral: {
+      variant: 'ConnectivityNeutral',
+      children: 'Neutral',
+    },
+    Offline: {
+      variant: 'ConnectivityOffline',
+      children: 'Offline',
+    },
+  };
+  const [availability, setConnectivity] = React.useState(
+    ConnectivityObject.Available
+  );
+  const menu = useStatusMenuState();
+  const supportMenu = useMenuState();
+  const onClick = (status) => {
+    setConnectivity(ConnectivityObject[status]);
+    menu.hide();
+  };
+  const productSwitcher = useProductSwitcherState();
+  const [product, setProduct] = React.useState('twilio');
+  const userDialogList = useUserDialogListState();
+
+  return (
+    <Box minWidth="size70">
```

Comment:
thought: should we remove this Box or adjust the minWidth down? On my laptop at fullscreen my examples overflow. See the bottom example here:

<img width="750" alt="image" src="https://github.com/twilio-labs/paste/assets/1592327/b8888b61-17bf-4201-a3d3-d6625b812e3d">